pom.xml

```diff
@@ -175,6 +175,24 @@
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-job</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-job</artifactId>
+      <version>2.39</version>
+      <scope>compile</scope>
```

Comment:
As commented somewhere before, this is a bad idea to use compile scope

Reply:
Fixed